public/main/group/group_creation.php

```diff
@@ -325,14 +347,42 @@ function copy_value(key) {
         $classForm->addHeader(get_lang('Groups from classes'));
 
         $classForm->addHtml($description);
+
+        $classGroup = [];
+
+        foreach ($classes as $index => $class) {
+            $number_of_users = count($obj->get_users_by_usergroup($class['id']));
+            // $classForm->addCheckBox('checkbox_class_id_'.$class['id'], $class['title'] . ' ('.$number_of_users.' '.get_lang('Users').')');
+            $classGroup[] = $classForm->createElement('checkbox', 'checkbox_class_id_'.$class['id'], null, $class['title'] . ' ('.$number_of_users.' '.get_lang('Users').')');
+        }
+
+        $classForm->addGroup(
+            $classGroup,
+            '',
+            null,
+            null,
+            false
+        );
+
         $classForm->addElement('hidden', 'action');
         if ($allowGroupCategories) {
-            $classForm->addSelect('group_category', null, $categories);
+            if (isset($_GET['category_id'])) {
+                $classForm->addElement('hidden', 'group_category', $_GET['category_id']);
+            } else {
+                $classForm->addSelect('group_category', null, $categories);
+            }
         } else {
             $classForm->addElement('hidden', 'group_category');
         }
+
+        $classForm->addHtml('<div style="height: 1px; width: 100%; background-color: #e6e6e6; border-radius: 8px;" /><br>');
+
+        $classForm->addCheckBox('is_consistent_link', null, get_lang('Link classes to created groups ?'),
+            ['title' => get_lang('Info message about deactivation of consistent link')]
```

Comment:
An info message should not start with "Info message about". It should be directly the message to the user :smile: